### PR TITLE
Comments App

### LINKED ISSUE #169: Comments App
Final round of work on the comments app for 0.3.

### PR DESCRIPTION
**Pull request by fastlinemedia**
Resolves #169

### CODE REVIEW

File: src/system/ui/lists/style.scss [hunk 1]
```diff
@@ -225,6 +225,11 @@
 		.fl-asst-is-favorite {
 			color: var(--fl-asst-bookmark-color);
 		}
+	}
+
+
+	.fl-asst-comment-extras {
+        display: block !important;
```
Comment (fastlinemedia) [inline src/system/ui/lists/style.scss]: These styles should probably be with the relevant component or inside the app if specific only to the app.

File: src/system/ui/pages/comment/actions.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+		editUrl,
+		approved,
+		spam,
+		trash,
+	} = item
+	const comments = getWpRest()
+	const { setCurrentHistoryState } = getSystemActions()
+	const [ actionApprove, set_actionApprove ] = React.useState( approved )
+	const [ actionSpam, set_actionSpam ] = React.useState( spam )
+	const [ actionTrash, set_actionTrash ] = React.useState( trash )
```
Comment (fastlinemedia) [inline src/system/ui/pages/comment/actions.js]: Variables should be camel case, even state setters.

File: src/apps/fl-comments/tabs/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import { List, App, Page } from 'assistant/ui'
+
+
+export const AllTab = () => {
+
+	const { handle } = useContext( App.Context )
+
+	return (
+		<>
+
```
[the remark this answers is not in the document]
Comment (shahidajmeri786) [reply src/apps/fl-comments/tabs/index.js]: Removed space. let me know if you find in other app.

File: src/system/ui/appearance/style.scss [hunk 4]
```diff
@@ -245,5 +245,23 @@
...
+	}
+	.CmtupdtBtn{
+		float: right;
+		margin-top: 10px;
+	}
+	.CmtTextWrap{
+		float:left;
+		width: 100%;
+		margin-top: 10px;
+	}
```
[the remark this answers is not in the document]
Comment (shahidajmeri786) [reply src/system/ui/appearance/style.scss]: Moved styles to comment app style file.

File: src/system/ui/forms/style.scss [hunk 5]
```diff
@@ -240,5 +240,14 @@
...
+	}
+
+	.comment-text {
+		padding: 15px;
+	}
+	.edit-comment-title{
+		font-weight: 600;
+    	margin-bottom: 10px;
+    	float: left;
+	}
```
[the remark this answers is not in the document]
Comment (shahidajmeri786) [reply src/system/ui/forms/style.scss]: Moved styles to comment app style file.

File: src/system/ui/lists/comments/index.js [hunk 6]
```diff
@@ -1,23 +1,212 @@
...
 	getItemProps = ( item, defaultProps ) => defaultProps,
+	type = { type },
 	query = {
-		commentStatus: 'all',
+		status: type,
 	},
 	...rest
 } ) => {
+	const comments = getWpRest()
+	const { setCurrentHistoryState } = getSystemActions()
```
[the remark this answers is not in the document]
Comment (shahidajmeri786) [reply src/system/ui/lists/comments/index.js]: I thought we need it but now are changing icons based on the state so now no need it anymore.

File: src/system/ui/lists/style.scss (hunk 1 above)
Comment (shahidajmeri786) [reply src/system/ui/lists/style.scss]: Moved styles to comment app style file.

File: src/system/ui/lists/comments/index.js [hunk 7]
```diff
@@ -1,23 +1,212 @@
...
-
+import { List, Button, Icon } from 'ui'
+import { truncate } from 'utils/text'
+import { __ } from '@wordpress/i18n'
+import { getSystemActions } from 'data'
+import { getWpRest } from 'utils/wordpress'
 export const Comments = ( {
+
 	getItemProps = ( item, defaultProps ) => defaultProps,
+	type = { type },
```
[the remark this answers is not in the document]
Comment (shahidajmeri786) [reply src/system/ui/lists/comments/index.js]: Yes it is passing 'all' when no type passed

File: src/system/ui/pages/comment/actions.js (hunk 2 above)
Comment (shahidajmeri786) [reply src/system/ui/pages/comment/actions.js]: Ok! Renamed variables to camel case.

**Pull request merged**